src/vue/modules/create-atomic-swap-form/components/create-atomic-swap-form.vue

```diff
@@ -313,6 +364,38 @@ export default {
       // eslint-disable-next-line max-len
       return base.CreateAtomicSwapAskRequestBuilder.createAtomicSwapAskRequest(operation)
     },
+
+    async postIssuanceOperation () {
+      const operation = base.CreateIssuanceRequestBuilder
+        .createIssuanceRequest({
+          asset: this.form.asset.code,
+          amount: this.getIssuanceAmount(),
+          receiver: this.accountBalance.id,
+          reference: this.getReference(),
+          creatorDetails: {},
+        })
```

Comment:
the operation should be posted in a single transaction with atomic swap operation, but placed _before_ the atomic swap operation,